server/routes/meetups.js

```diff
@@ -3,19 +3,20 @@
 /* eslint-disable eol-last */
 /* eslint-disable import/first */
 import express from 'express';
-import validate from 'express-validation';
 
 const router = express.Router();
 
+
 import meetupController from '../controller/meetups';
 import rsvpController from '../controller/rsvp';
-import validations from '../middleware/validator/validator';
 import Auth from '../middleware/Auth';
+import meetupValidation from '../middleware/validatemeetups';
 
-router.post('/', Auth.verifyToken, Auth.adminAuth, validate(validations.createMeetup), meetupController.createMeetup);
+router.post('/', Auth.verifyToken, Auth.adminAuth, meetupValidation.validCreateMeetup, meetupValidation.checkDate, meetupController.createMeetup);
 router.get('/upcoming/', Auth.verifyToken, meetupController.getUpcomingMeetups);
-router.get('/:meetupId', Auth.verifyToken, validate(validations.getMeetup), meetupController.getUpcomingMeetups);
+router.get('/:id', Auth.verifyToken, meetupController.getSingleMeetup);
```

Comment:
Since `Auth.verifyToken` is applied to all the routes here, why not move it to where this route file is mapped in the index.js file to avoid repetition?

Reply:
OK, would do that